packages/vanilla-extract/stories/Text.ts

```diff
@@ -24,3 +24,17 @@ export const ResponsiveThemedText = ({ text, background }: Props) =>
   `<div class="${styles.fontFamily} ${styles.responsiveThemedText}"${
     background ? ` style="background: ${background}"` : ''
   }>${text}</div>`;
+
+export const ContainerResponsiveText = ({ text, background }: Props) =>
+  [350, 550, 700]
+    .map(
+      (width) => `
+      <div style="width: ${width}px;" class="${styles.container}">
```

Comment:
These should probably use a `max-width`:
```suggestion
      <div style="width: 100%; max-width: ${width}px;" class="${styles.container}">
```

At the moment they seem to just grow and never conditionally adapt.